docs/README_ESP.md

```diff
@@ -0,0 +1,135 @@
+![image](https://user-images.githubusercontent.com/6539258/154814361-066e5010-40a8-4820-a706-ed9803a319a1.png)
+
+<div align="center">
+⚠️ ESTE SOFTWARE TODAVÍA ESTÁ EN ALPHA Y PUEDE CONTENER BUGS ⚠️
+</div>
+  
+# Para usuarios
+
+ODeck es una alternativa gratuita y de código abierto a StreamDeck hecha con React, React Native, Electron, Socket.IO y algunas otras tecnologías. Le permite controlar fácilmente algunas funciones de su computadora con el poder de un toque en su teléfono inteligente.
+
+Suele ser una alternativa más económica a StreamDeck porque puedes usar tu smartphone actual o limpiar el polvo de un dispositivo antiguo del fondo de tu cajón y darle otro propósito de vida a este dispositivo.
+
+## SO compatibles
+
+Actualmente, ODeck aún no es compatible con todas las plataformas, pero debería serlo en un futuro próximo. Consulte la lista de los sistemas operativos compatibles actualmente.
+
+### Deck
+
+- Android
+
+### Panel de administración
+
+- Windows
+
+## Características actuales
+
+- Abrir una URL
+- Emite una combinación de teclas para invocar teclas de acceso rápido
+- Ejecutar un archivo EXE
+- Teclas de medios para interactuar con los medios que se reproducen en su sistema
+- Crear varias páginas
+- Reproducir cualquier sonido al presionar un botón
+- Navegar entre páginas
+- Empareje la aplicación con el código QR
+
+## Próximas características
+
+- Integración con OBS
+- Input de carga de imagen
+- Evitar el diseño roto con una cantidad de teclas personalizadas
+- Integración con Twitch
+- Agregar compatibilidad con Linux y MacOS para el panel de administración
+- Agregar compatibilidad con iOS para la aplicación ODeck
+
+### Vista previa del deck
+
+https://user-images.githubusercontent.com/6539258/154787245-872187a9-b05e-44c4-b162-42037ace352a.mp4
+
+### Vista previa del panel de administración
+
+https://user-images.githubusercontent.com/6539258/154787603-3cf7bcd0-f82c-4092-96b9-28546fe16568.mp4
```

Comment:
I updated the readme to remove the media files that was in the `.media` directory, and use the new github  repository assets.

```suggestion
### Vista previa del deck

https://github.com/willianrod/ODeck/assets/6539258/9c58dbb1-38f6-4602-b211-aecbfc8e4dff

### Vista previa del panel de administración

https://github.com/willianrod/ODeck/assets/6539258/51710202-d665-4f8c-bd7c-17c470e8e29b
```